Project/js/filter.js

```diff
@@ -0,0 +1,15 @@
+let list = document.getElementsByClassName("cities-list__element");
+
+function search() {
+    let input = document.getElementsByClassName("content-search__field")[0].value.toLowerCase();
```

Comment:
since you are using `.content_search_field` more than once, it'd be better to cache it at the beginning of the file together with the `list`. Also, make sure you are not using `let` where `const` would do.